src/app/search/search.component.html

```diff
@@ -10,7 +10,8 @@ <h5 class="mdl-dialog__title" style="font-size: 15px;margin-left: 14px;">Edit yo
   </dialog>
   <ng-container *ngTemplateOutlet="formdata"></ng-container>
   <div class="searchResults__mapListView">
-    <map-view [locationData]="applicableLocations" class="map"></map-view>
+    <map-view *ngIf="this.useMap" [locationData]="applicableLocations" class="map"></map-view>
+    <div *ngIf="!this.useMap" id="dummyMap"></div>
```

Comment:
Please use ng-template. take reference from home page where we are loading popular destinations

Reply:
Done with the changes. Thanks!